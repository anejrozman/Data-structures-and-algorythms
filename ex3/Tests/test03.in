11,26,3
2.4249013400157593,0.6190497077986656,7.845508726821936,4.240559921901058,3.7511685593302033,1.0091413662526172,3.3332664450901333,1.8917236878317276,4.501619062414473,6.31897318777646,6.062573395442523
2,8,898.144
2,9,386.87
0,6,133.856
8,0,902.811
10,6,391.31
2,4,138.239
7,0,780.048
2,0,22.735
8,4,663.924
9,5,803.903
7,3,935.338
7,8,684.137
7,1,689.594
4,1,696.267
5,6,957.067
3,1,318.466
3,9,452.843
2,7,708.347
5,4,709.03
2,10,971.913
1,6,204.805
0,10,722.261
9,4,868.303
1,10,486.066
10,5,881.907
6,2,639.625
