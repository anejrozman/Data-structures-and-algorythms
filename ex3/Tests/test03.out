9,660.3477
1,0
1,0
